### DISCUSSION TITLE
The UI is amazing as well as the feature set.

### DISCUSSION THREAD

**Discussion by ServerGeek82** [JordanKnott/taskcafe#162] [Q&A]
But is the project abandoned or backlogged?

**Comment by JordanKnott** [comment]
Hi the project is not abandoned - but is on temporary hold while I deal with some personal issues.

But it is still being slowly worked on.

**Reply by ServerGeek82** [reply]
Fields and Rules are not available. Will this be added in next release?

**Reply by GwynethLlewelyn** [reply]
> [...] while I deal with some personal issues.

I hope you're all right, and wish you good luck with those!

**Comment by ServerGeek82** [comment]
Awesome. Looking forward to what's next.
